Sources/YCoreUI/Extensions/UIKit/UIView+constrainCenter.swift

```diff
@@ -0,0 +1,71 @@
+//
+//  UIView+constrainCenter.swift
+//  YCoreUI
+//
+//  Created by Dharmik Ghelani on 20/10/22.
+//  Copyright © 2022 Y Media Labs. All rights reserved.
+//
+
+import UIKit
+
+extension UIView {
+    /// Center alignment options
+    public struct Center: OptionSet {
+        public let rawValue: UInt
```

Comment:
So this needed to be made public after
```suggestion
        /// corresponding raw value
        public let rawValue: UInt
        
```

Because it's now public, we have to document it.
Let's also add a single vertical whitespace between this and the static values.

Reply:
Addressed